jpo-ode-svcs/src/test/java/us/dot/its/jpo/ode/OdeTimJsonTopologyTest.java

```diff
@@ -5,35 +5,33 @@
 import org.apache.kafka.streams.Topology;
 import org.apache.kafka.streams.state.ReadOnlyKeyValueStore;
 import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import us.dot.its.jpo.ode.kafka.OdeKafkaProperties;
+
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.jupiter.api.Assertions.assertTrue;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.Mockito.doNothing;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
-import us.dot.its.jpo.ode.kafka.OdeKafkaProperties;
-
 
-public class OdeTimJsonTopologyTest {
+class OdeTimJsonTopologyTest {
 
     private OdeTimJsonTopology odeTimJsonTopology;
     private KafkaStreams mockStreams;
     private ReadOnlyKeyValueStore<String, String> mockStore;
-    private OdeProperties mockOdeProps;
     private OdeKafkaProperties mockOdeKafkaProps;
 
     @BeforeEach
-    public void setUp() throws NoSuchFieldException, SecurityException, IllegalArgumentException, IllegalAccessException {
-        mockOdeProps = mock(OdeProperties.class);
+    public void setUp() throws SecurityException, IllegalArgumentException {
```

Comment:
Does this setUp() method need to be public?

Reply:
It doesn't. good eye. updated here https://github.com/CDOT-CV/jpo-ode/pull/112/commits/ddebb8f259f2d6a5f736c378e4e3c58068c9b0f7